tests/console/docker_image_rpm.pm

```diff
@@ -29,12 +29,10 @@ sub run {
     install_docker_when_needed();
 
     if (is_tumbleweed) {
-        if (check_var('ARCH', 'aarch64')) {
-            $repo_url = 'https://download.opensuse.org/repositories/Virtualization:containers:images:openSUSE-Tumbleweed/container_ARM/Virtualization:containers:images:openSUSE-Tumbleweed.repo';
-        }
-        else {
-            $repo_url = 'https://download.opensuse.org/repositories/Virtualization:containers:images:openSUSE-Tumbleweed/container/Virtualization:containers:images:openSUSE-Tumbleweed.repo';
-        }
+        my $defname = "container";
```

Comment:
I would use ternary operator or elsif here to avoid 2 assignments and checking ARCH variable twice even if we already matched arm condition, e.g. :
```perl
    my $defnme =
      check_var('ARCH', 'aarch64')  ? 'container_ARM'
      : check_var('ARCH', 'ppc64le') ? 'container_PowerPC'
      :                                                   'container';
```